scrapers/data18.yml

```
@@ -2,34 +2,24 @@ name: data18
 sceneByURL:
   - action: scrapeXPath
     url:
-      - data18.com/content
+      - data18.com/scenes
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
     common:
-      $performer: //p[b[text()='Starring:']]/a
-      $studio: //p[b[text()='Site:' or text()='Studio:']]/a
+      $performer: //b[text()='Pornstars / Cast']/../..//a[@class='bold gen']
```

In a few scenes the performer doesnt have a page so we get a span instead of an anchor
```suggestion
      $performer: //b[text()='Pornstars / Cast']/../..//*[local-name()='a' or local-name()= 'span'][@class='bold gen']
```
It shouldnt affect the URL as we will get none in that case